### ISSUE TITLE
Review landing page revisions

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/PDS.nasa.gov-Search#28]
Review updates from #27

**Comment by tloubrieu-jpl** [comment]
@jordanpadams @edwardbarraza do you have an update from @mdrum ?

**Comment by mdrum** [comment]
Yep: Here's where things are now: 
![CleanShot 2020-11-12 at 16 57 58@2x](https://user-images.githubusercontent.com/31220657/99011095-8b2ed380-2508-11eb-925b-bbb89369bbd6.png)

I plan to deploy these changes tomorrow, and we can go over them in the meeting.

**Comment by jordanpadams** [comment]
@tloubrieu-jpl we will be reviewing this at our tag-up this afternoon

**Comment by jordanpadams** [comment]
comments from meeting on 11/13/2020:

design system improvements needed:
* update design system to explicitly call out background color
* specify the font sizes as responsive values

updates to latest iteration:
* PDS4 collection tag at top - may eventually be changed, but don't worry about it right now
* and others...

**Comment by mdrum** [comment]
@jordanpadams I've completed and deployed the feedback changes that we discussed on Friday: https://sbnarchivedemo.psi.edu/?identifier=urn:nasa:pds:cassini_iss_saturn:data_raw&mockup=true

I believe this should be ready for demo/screenshots. If you see anything else you'd like me to tweak just let me know!

(As a reminder, for mockup data/features to be shown on the page, make sure `&mockup=true` is in the URL. If you'd also like to hide any supplemental metadata from a given page, then also append `&pdsOnly=true`)
